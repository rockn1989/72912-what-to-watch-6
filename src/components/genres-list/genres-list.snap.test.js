import React from 'react';
import {render} from '@testing-library/react';
import GenresList from './genres-list';

it(`Should GenresList render correctly`, () => {
  const data = {
    genre: `Drama`,
    setGenre: () => {},
    genresList: [`Comedy`, `Drama`]
  };

  const {container} = render(<GenresList {...data}/>);
  expect(container).toMatchSnapshot();
});
