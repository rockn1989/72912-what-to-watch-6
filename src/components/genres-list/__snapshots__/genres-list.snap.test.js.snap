// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should GenresList render correctly 1`] = `
<div>
  <ul
    class="catalog__genres-list"
  >
    <li
      class="catalog__genres-item "
    >
      <a
        class="catalog__genres-link"
        href="#"
      >
        Comedy
      </a>
    </li>
    <li
      class="catalog__genres-item catalog__genres-item--active"
    >
      <a
        class="catalog__genres-link"
        href="#"
      >
        Drama
      </a>
    </li>
  </ul>
</div>
`;
